services-in-london/src/App.js

```diff
@@ -1,12 +1,78 @@
 import React, { Component } from 'react';
 import './App.css';
 import Area from './area';
-import Organization from './organization';
+import Type from './type';
+import Clients from './clients';
 
 class App extends Component {
 	constructor(props) {
 		super(props);
 		this.state = {
+				main: ['area','type','clients'],
+				clickedValue: null,
+				callData: null,
+		}
+	}
+	servicesBy = (info) =>{
+		this.setState({clickedValue: info});
+		this.componentDidMount(info);
+	}
+	displayRender = () => {
+		switch (this.state.clickedValue) {
+			case 'clients': return (<Clients dataInfo={this.state.callData} />);
+			break;
+			case 'type': return (<Type dataInfo={this.state.callData} />);
+			case 'area': return (<Area dataInfo={this.state.callData} />);
+			default:
+			break;
+		}
+		/*let test = '';
```

Comment:
You should not need to include comments in code, as your git commits will keep the old ways you did it so you can always view them in your git history. Best to use comment just to add information/clarity rather than old code